### DISCUSSION TITLE
Estadisticas columnas

### DISCUSSION THREAD

**Discussion by romu545** [posit-dev/positron#3954] [Feedback]
Sugiero colocar las estadísticas dentro de las respectivas columnas. Esto para evitar el layuot de las variables.
![Captura de pantalla 2024-07-10 112333](https://github.com/posit-dev/positron/assets/76394912/526c43fc-e686-4dba-b354-f6fc97e220e5)

**Comment by juliasilge** [comment]
Thanks for this feedback! We are continuing to iterate on the Data Explorer and especially that column summary panel.
